third_party/2and3/redis/client.pyi

```diff
@@ -31,16 +34,36 @@ def parse_hscan(response, **options): ...
 def parse_zscan(response, **options): ...
 def parse_slowlog_get(response, **options): ...
 
-class StrictRedis:
+class Redis(object):
     RESPONSE_CALLBACKS: Any
     @classmethod
     def from_url(cls, url, db=..., **kwargs): ...
     connection_pool: Any
     response_callbacks: Any
-    def __init__(self, host=..., port=..., db=..., password=..., socket_timeout=..., socket_connect_timeout=...,
-                 socket_keepalive=..., socket_keepalive_options=..., connection_pool=..., unix_socket_path=..., encoding=...,
-                 encoding_errors=..., charset=..., errors=..., decode_responses=..., retry_on_timeout=..., ssl=...,
-                 ssl_keyfile=..., ssl_certfile=..., ssl_cert_reqs=..., ssl_ca_certs=...) -> None: ...
+    def __init__(
+        self,
+        host: Text = ...,
+        port: int = ...,
+        db: int = ...,
+        password: Optional[Text] = ...,
+        socket_timeout: Optional[float] = ...,
+        socket_connect_timeout: Optional[float] = ...,
+        socket_keepalive: Optional[bool] = ...,
+        socket_keepalive_options: Optional[Mapping[str, Union[int, str]]] = ...,
+        connection_pool: Optional[ConnectionPool] = ...,
+        unix_socket_path: Optional[Text] = ...,
+        encoding: Text = ...,
+        encoding_errors: Text = ...,
+        charset: Optional[Text] = ...,
+        errors: Optional[Text] = ...,
+        decode_responses: bool = ...,
+        retry_on_timeout: bool = ...,
+        ssl: bool = ...,
+        ssl_keyfile: Optional[Text] = ...,
+        ssl_certfile: Optional[Text] = ...,
+        ssl_cert_reqs: Optional[Union[str, int]] = ...,
+        ssl_ca_certs: Optional[Text] = ...,
```

Comment:
Missing the `max_connections=None` argument (https://github.com/andymccurdy/redis-py/blob/master/redis/client.py#L635).